client/components/overview/ImageGallery.jsx

```diff
@@ -1,20 +1,70 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import ImageNav from './ImageNav.jsx'
+import {Grid} from '@material-ui/core';
```

Comment:
I recommend changing your material ui import statements to utilize Tree-Shaking: https://material-ui.com/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking